app/src/components/inputs/inputChekbox/inputCheckbox.jsx

```diff
@@ -18,56 +18,62 @@
  * You should have received a copy of the GNU General Public License
  * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
  */
-
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import styles from './inputCheckbox.scss';
 
 const cx = classNames.bind(styles);
 
-export const InputCheckbox = ({ children, value, disabled, onChange, onFocus, onBlur }) => {
-  const squareClasses = cx({
-    square: true,
-    centered: !children,
-    checked: value,
-    disabled,
-  });
-  const handlerOnChange = (e) => {
-    onChange({ value: e.target.checked });
+export class InputCheckbox extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    value: PropTypes.bool,
+    disabled: PropTypes.bool,
+    onChange: PropTypes.func,
+    onFocus: PropTypes.func,
+    onBlur: PropTypes.func,
+  };
+  static defaultProps = {
+    children: '',
+    value: false,
+    disabled: false,
+    onChange: () => {},
+    onFocus: () => {},
+    onBlur: () => {},
+  };
+  state = {
+    checked: this.props.value,
+  };
+  handlerOnChange = (e) => {
+    this.setState({ checked: e.target.checked });
+    this.props.onChange({ checked: e.target.checked });
   };
-  return (
-    <label className={cx('input-checkbox')} onFocus={onFocus} onBlur={onBlur} tabIndex="1">
-      <input
-        type="checkbox"
-        className={cx('input')}
-        checked={value}
-        disabled={disabled}
-        onChange={handlerOnChange}
-      />
-      <div className={squareClasses}>
-        <svg className={cx('icon')} xmlns="http://www.w3.org/2000/svg" width="9" height="10" viewBox="0 0 8 7">
-          <polygon fill={disabled ? '#999' : '#fff'} fillRule="evenodd" points="0 3.111 3 6.222 8 1.037 7 0 3 4.148 1 2.074" />
-        </svg>
-      </div>
-      <span className={cx({ 'children-container': true, disabled })}>{children}</span>
-    </label>
-  );
-};
+  render() {
+    const squareClasses = cx({
+      square: true,
+      centered: !this.props.children,
+      checked: this.state.checked,
+      disabled: this.props.disabled,
+    });
+    return (
+      <label className={cx('input-checkbox')} onFocus={this.props.onFocus} onBlur={this.props.onBlur} tabIndex="1">
+        <input
+          type="checkbox"
+          className={cx('input')}
+          checked={this.state.checked}
+          disabled={this.props.disabled}
+          onChange={this.handlerOnChange}
+        />
+        <div className={squareClasses}>
+          <svg className={cx('icon')} xmlns="http://www.w3.org/2000/svg" width="9" height="10" viewBox="0 0 8 7">
```

Comment:
I think svg worth to be extracted as separate component with `className` and `disabled` props